third_party/ccid/webport/build/Makefile

```diff
@@ -47,6 +48,16 @@ CCID_NACL_SOURCES_PATH := ../src
 #   for all PC/SC-Lite server drivers; use a relative path, so that it works
 #   both inside the Smart Card Connector app and in unit tests;
 COMMON_CPPFLAGS := \
+	-DIFDHCloseChannel=CCID_IFDHCloseChannel \
```

Comment:
To note, I was looking for a more elegant way of achieving this, and the `objcopy --redefine-sym` seemed promising, however Wasm turned out to be unsupported in it (https://github.com/llvm/llvm-project/issues/50623)...